src/index.js

```diff
@@ -1,39 +1,33 @@
-import home from './home.js'
-import tabCreation from './tab.js'
-import about from './about.js'
-import contact from './contact.js'
-
-
-
-window.onload = function () {
-    const content = document.getElementById("content")
-
-    content.appendChild(tabCreation());
-
-    const homeTab = document.getElementById("home");
-    const aboutTab = document.getElementById("about");
-    const contactTab = document.getElementById("contact");
-
-
-    homeTab.addEventListener('click', (e) => {
-        deletePreviousContent(content);
-        content.appendChild(home());
-    })
-    aboutTab.addEventListener('click', (e) => {
-        deletePreviousContent(content);
-        content.appendChild(about());
-    })
-    contactTab.addEventListener('click', (e) => {
-        deletePreviousContent(content);
-        content.appendChild(contact());
-    })
-};
-
+import home from './home';
+import tabCreation from './tab';
+import about from './about';
+import contact from './contact';
 
 const deletePreviousContent = (parent) => {
-    while (parent.lastChild && parent.children.length > 1) {
-        console.log(parent.lastChild)
-        parent.removeChild(parent.lastChild);
-    }
-}
+  while (parent.lastChild && parent.children.length > 1) {
+    parent.removeChild(parent.lastChild);
+  }
+};
 
+window.onload = () => {
+  const content = document.getElementById('content');
+
+  content.appendChild(tabCreation());
+
+  const homeTab = document.getElementById('home');
+  const aboutTab = document.getElementById('about');
+  const contactTab = document.getElementById('contact');
+
+  homeTab.addEventListener('click', () => {
+    deletePreviousContent(content);
+    content.appendChild(home());
+  });
```

Comment:
- [ ] [REQUIRED] Each tab module should be able to append itself to the DOM, right now the index file is the one appending the tabs to the DOM
![image](https://user-images.githubusercontent.com/73254040/101965731-4ebfc780-3bdb-11eb-9ce5-ecbedbcfbca4.png)
[Source](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)